src/pydargs/__init__.py

```diff
@@ -46,101 +46,102 @@ def _create_parser(tp: Type[Dataclass]) -> ArgumentParser:
         if field.metadata.get("ignore_arg", False):
```

Comment:
are we adding the positionals in the order of the fields? is `fields` always ordered in the same way?

Reply:
Fields are ordered in the order of definition inside the dataclass. So we are adding them in that order.